<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FomCreateUserDialog</class>
 <widget class="QDialog" name="FomCreateUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nomLabel">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nomLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dateDInscriptionLabel">
      <property name="text">
       <string>Date d'inscription</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="dateDInscriptionLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="sexeLabel">
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="sexeLineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> Ajouter un membrer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
